contrib/utilities/jsontomarkdown.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python3
+
+import json
+from sys import argv, exit, stderr
+
+def handle_subsection(data, cur_path, output_file):
+    keys = list(data.keys())
+    keys.sort()
+    # Since there is no end to a markdown heading besides another markdown
+    # heading of equal or greater size, we need to first do parameters and
+    # aliases, and then do subsections otherwise parameters get nested
+    # incorrectly
+    for key in keys:
+        path_str = "/".join(cur_path) + "/"  + key
+        true_name = key.replace("_20", " ")
+        
+        if "value" in data[key]:
+            # this is a parameter
+            print("(parameters:" + path_str + ")=", file=output_file)
+            print("### __Parameter name:__ " + true_name, file=output_file)
+            print("**Default value:** " +  data[key]["default_value"] + " \n", file=output_file)
+            print("**Pattern:** " +  data[key]["pattern_description"] + " \n", file=output_file)
+            print("**Documentation:** " + data[key]["documentation"] + " \n", file=output_file)
+        elif "alias" in data[key]:
+            # This is an alias for a parameter
+            print("(parameters:" + path_str + ")=", file=output_file)
+            aliased_name = data[key]["alias"]
+            alias_path_str = "/".join(cur_path) + "/" + aliased_name.replace(" ", "_20") 
+            print("### __Parameter name__: " +  true_name, file=output_file)
+            print("**Alias:** [" + aliased_name + "](parameters:" + alias_path_str + ")\n", file=output_file)
+            print("**Deprecation Status:** " + data[key]["deprecation_status"] + "\n", file=output_file)
```

Comment:
This is the same code as for function below, right? Could you refactor this into a function of its own so we don't duplicate things?